src/Query/Conditions/BetweenConditionBuilder.php

```diff
@@ -44,9 +52,11 @@ public function build(ExpressionInterface $expression, array &$params = []): str
      * @param mixed $value
      * @param array $params passed by reference
      *
-     * @return string
+     * @throws InvalidArgumentException|Exception|InvalidConfigException|NotSupportedException
+     *
+     * @return array|string
      */
-    protected function createPlaceholder($value, array &$params): string
+    protected function createPlaceholder($value, array &$params)
```

Comment:
That should always return a string or else code in `build()` will fail.

Reply:
fixed.